resources/disk/make_rootfs.sh

```diff
@@ -37,11 +37,12 @@ Options:
   -d debdir         Directory containing .deb packages for the Linux image.
   -w workdir        Working directory for the kernel build.
   -o diskfile       Name of the resulting disk file.
+  -f flavour        The Ubuntu flavour desired. Defaults to "focal"
```

Comment:
Focal describes the version of ubuntu, not the flavor. Can you rename this to version?

Reply:
Actually this commit does not pertain to this PR. I will make sure to update this in this one: #150.